web-app/packages/lib/src/modules/dashboard/views/DashboardViewTemplate.vue

```diff
@@ -8,12 +8,14 @@ SPDX-License-Identifier: AGPL-3.0-only OR LicenseRef-MerginMaps-Commercial
   <div>
     <slot name="usageInfo" />
     <slot name="content"></slot>
+    <community-banner />
```

Comment:
is this meant for all versions (CE/EE/cloud)?

Reply:
currently yes - it could be slot in future.